tests/Config/ConfigTest.php

```diff
@@ -40,6 +40,7 @@ public function testConfigHasApiUrls(): void
             [
                 Config::SERVICE_AUTHENTICATION => '/test/auth',
                 Config::SERVICE_AUDITS => '/audits',
+                Config::SERVICE_EVENTS => '/eventzz',
```

Comment:
?

Reply:
This is only for testing purposes for config. To test the ability to override default config settings.